homeassistant/components/flux_led/light.py

```diff
@@ -333,6 +331,12 @@ def turn_on(self, **kwargs):
         if brightness is None:
             brightness = self.brightness
 
+        if hs_color is None or (hs_color[0] == 0 and hs_color[1] == 0):
+            hs_color = (1, 1, 1)
+        
```

Comment:
hs_color will only be passed to flux_led when a change in color is made.  Also, flux_led checks to see if hs_color is passed on line 282.  If it did receive an hs_color then it converts it to RGB and effectively discards hs_color.  It isn't used anywhere else. 

The lines you added for hs_color are not doing anything.